package/submariner.io.lighthouse_serviceimportweightmap-crd.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: serviceimportweightmaps.lighthouse.submariner.io
+spec:
+  group: lighthouse.submariner.io
+  names:
+    kind: ServiceImportWeightMap
+    listKind: ServiceImportWeightMapList
+    plural: serviceimportweightmaps
+    singular: serviceimportweightmap
+  scope: Namespaced
```

Comment:
It doesn't seem like this needs to be namespaced. The code watches all namespaces anyway.